server/middleware/roleMiddleware.ts

```diff
@@ -1,14 +1,15 @@
 import { NextFunction, Request, RequestHandler, Response } from 'express'
 import { TemporaryAccommodationUserRole as UserRole } from '../@types/shared'
 import { UnauthorizedError } from '../utils/errors'
+import { hasRegionEnabled } from '../utils/userUtils'
 
 export const createRoleMiddleware = (role: UserRole) => {
   return (handler: RequestHandler) => wrapHandler(handler, role)
 }
 
 const wrapHandler =
   (handler: RequestHandler, role: UserRole) => async (req: Request, res: Response, next: NextFunction) => {
-    if (res.locals.user.roles.includes(role)) {
+    if (res.locals.user.roles.includes(role) && hasRegionEnabled(res.locals.user)) {
```

Comment:
I think this needs a a rethink, as AFAICT this will end up restricting the entire app to users in our enabled regions. 

Tempted to say that the signature for `wrapHandler` should be:
`const wrapHandler = (handler: RequestHandler, check: (user: User) => bool))`...

Then instead of doing:
`if (res.locals.user.roles.includes(role))`
It does:
`if (check(res.locals.user)`

And createRoleMiddleware passes in a function, but even then I think you'd need to split out our routes for this to not break things for someone.

It may be we just don't do enforcement at this level